airbyte-workers/Dockerfile

```diff
@@ -20,5 +20,11 @@ WORKDIR /app
 # Move worker app
 ADD bin/${APPLICATION}-${VERSION}.tar /app
 
+# Grab well-known types file
+RUN wget https://github.com/mikefarah/yq/releases/download/v4.30.4/yq_linux_${DOCKER_BUILD_ARCH}.tar.gz -O - | tar xz \
```

Comment:
nit: This could be done earlier for a better caching of the steps. Probably around line 10.

Reply:
removed this entirely since the file gets generated in gradle